spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingServiceConfiguration.java

```diff
@@ -83,6 +85,9 @@
 	@Autowired
 	private MessageBuilderFactory messageBuilderFactory;
 
+	@Lazy @Autowired(required = false)
```

Comment:
But a lazily referenced `CompositeMessageConverterFactory ` doesn't necesarily mean that the `ObjectMapper` it is injected with should be a lazy reference as well.